src/Microsoft.DotNet.Interactive.CSharp.Tests/CSharpKernelTests.cs

```diff
@@ -38,5 +38,19 @@ public async Task Script_state_is_available_within_middleware_pipeline()
             variableCountBeforeEvaluation.Should().Be(0);
             variableCountAfterEvaluation.Should().Be(1);
         }
+
+        [Fact]
+        public async Task GetValueInfos_only_returns_newly_bindable_objects()
```

Comment:
The name is a little confusing. Does this mean it does not return variables that have been shadowed?

Reply:
Yes, that is correct.  I'll update the name.